<?xml version= "1.0" ?>
<launch>
    <param name = "avg_win" value = "5"/>
    <node pkg="assignment_2_2023" type="last_target.py" name="last_target"  />
    <node pkg="assignment_2_2023" type="get_dist_avg_speed.py" name="Dist_AvgSpeed_Service"  />
</launch>
